packages/story-editor/src/components/library/panes/shopping/shoppingPane.js

```diff
@@ -235,9 +244,22 @@ function ShoppingPane(props) {
 
   const handleClearInput = useCallback(() => {
     setSearchTerm('');
-    debouncedProductsQuery('', orderby, order);
+    setPage(1);
+    setLoaded(false);
+    debouncedProductsQuery('', 1, orderby, order);
```

Comment:
Maybe move `setLoaded` above `setPage` here or use to keep passing the page number closer together ... not hard to spot but could get missed.

Reply:
Moved the setPage all together here.